Clarified language about what external drivers are in the context of … (#22633)
qmk/qmk_firmware: docs/feature_led_matrix.md

# LED Matrix Lighting :id=led-matrix-lighting

> This feature allows you to use non-RGB LED matrices driven by external drivers. *External* here means *not within the microcontroller,* and includes LEDs with embedded drivers. LED Matrix Lighting hooks into the backlight system so you can use the same keycodes as backlighting to control it.

Reviewer: ```suggestion
This feature allows you to use LED matrices controlled by external LED driver ICs.
```
